### DISCUSSION TITLE
Como ser um Programador Melhor  [LIDO] ✅

### DISCUSSION THREAD

**Discussion by lincolixavier** [pilhacheia/clube-do-livro-dev#10] [Books]
**Como ser um Programador Melhor: um Manual Para Programadores que se Importam com Código**

Se você é apaixonado por programação e quer se aperfeiçoar nisso, está com a fonte de informações perfeita. Pete Goodliffe, autor de Code Craft, apresenta um conjunto de técnicas e abordagens úteis para a arte da programação que irá ajudá-lo a impulsionar a sua carreira e a melhorar o seu bem-estar.
Goodliffe apresenta conselhos sólidos, aprendidos em 15 anos de programação profissional. Os capítulos independentes do livro cobrem o espectro da vida de um desenvolvedor de software – lidar com código, conhecer os negócios e melhorar o desempenho – sem ideias tendenciosas sobre qualquer linguagem ou mercado. Independentemente de ser um desenvolvedor experiente, um profissional novato ou um programador por hobby, você encontrará dicas valiosas

Preço: R$58,24
[Link amazon](https://www.amazon.com.br/Como-Ser-Um-Programador-Melhor/dp/8575224158)

**Comment by gustavoferrazfontes** [comment]
Wow! Achei bem interessante esse em! O tema é algo que gosto bastante, livro muito bem classificado na Amazon! Fiquei com vontade de ler.
